src/plugins/intel_gna/src/transformations/remove_pre_post_processing.cpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/remove_pre_post_processing.hpp"
+#include "transformations/utils/transformation_helper.hpp"
+#include <openvino/cc/ngraph/itt.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/opsets/opset7.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
```

Comment:
Why some of the includes from openvino use < and some "? Please check in other modified files as well.

Reply:
fixed